src/viewProviders/newResources.ts

```diff
@@ -444,7 +466,7 @@ export class NewResourceViewProvider
       // Always do a deep refresh of this connection.
       await connectionRow.refresh(deepRefresh);
 
-      this.logger.debug("Connection row refreshed, signaling row repaint.");
+      this.logger.debug("Connection row for refreshed, signaling row repaint.", { connectionId });
```

Comment:
Aside from removing the extra `for`, we shouldn't need any updates here since the logger includes the connection ID in its name, right?
https://github.com/confluentinc/vscode/blob/341a32c9ec4600eb54864f6c936d501c0fa2c0ca/src/viewProviders/newResources.ts#L100